umbraco-cloud/set-up/project-settings/dedicated-resources.md

```diff
@@ -39,4 +39,21 @@ Moving away from dedicated resources and back to shared can be done from the _De
 * By hitting "Downgrade to shared" and confirming your choice, you will be redirected to the project page where you will be notified when the move back to a shared resource has been completed.
 * Your Cloud project is now back on a shared resource.
 
-If you have any questions regarding dedicated resource, feel free to reach out to [Umbraco Support](mailto:contact@umbraco.com).
+
+## Frequently asked questions
+
+Wondering what happens when you move your environment? 
+Below you can find a list of the most frequently asked questions and the answers.
+
+* Will it move all the environments to the dedicated server?
+  * Yes, all environments will be moved to the dedicated server.
+* How will the resources be split betweeen the environments?
+  * All environments on the project moved to dedicated will share the ressources of the dedicated server.
+* Will the customer be able to work on the project during the move?
+  * It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the website will not respond as usual
```

Comment:
>It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the website will not respond as usual.

I would change this to Backoffice instead of website.

You can elaborate if you think that makes sense:
When its a move to dedicated within the same tier only the website resources are moved.
If its a move between Plans and to Dedicated, ie. from Starter to Standard Dedicated then the databases are also moved to a higher tier.
Because the website is moved and restarts and hostname switches happen everyone should refrain from working in the backoffice, because it might be switched over while someone is saving/publishing content.

Reply:
Makes sense with changing it to backoffice 👍

For now, I think it is fine to not go into too much detail about what happens behind the scenes.

We can always add it down the line if it is something people using cloud starts to ask more into it :)